app/views/nsm/steps/youth_court_claim_additional_fee/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<% title t('.page_title') %>
+<% decision_step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary(@form_object) %>
+    <h1 class="govuk-heading-l"><%= t('.heading') %></h1>
```

Comment:
nit: this is technically not the correct element to use, discovered this when doing my ticket after a discussion with Mary - the [GDS for radio buttons is formatted ](https://design-system.service.gov.uk/components/radios/) so that there is a h1 tag with class govuk-fieldset__heading within a legend tag. 

I don't think this should block deployment though as I think we have this problem in other places as well